jscomp/bsc/rescript_compiler_main.ml

```diff
@@ -273,6 +273,9 @@ let buckle_script_flags : (string * Bsc_args.spec * string) array =
     "-bs-ast", unit_call(fun _ ->  Js_config.binary_ast := true; Js_config.syntax_only := true),
     "*internal* Generate binary .mli_ast and ml_ast and stop";
 
+    "-code-action-data", unit_call(fun _ ->  Js_config.code_action_data := true),
+    "*internal* Emit code action data";
+
```

Comment:
This isn't actually used yet, but we can probably have a `bsc` config like this.